doc/usage.md

```diff
@@ -320,3 +320,61 @@ oc run --rm -i --restart=Never --image=registry.fedoraproject.org/fedora-minimal
 rer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" https://metrics.openshift-compliance.svc:8585/metrics-co' | grep compliance
 ```
 
+## To use PriorityClass for scans
+
+When heavily using Pod Priority and Preemption for automated scaling and
+the default PriorityClass is too low to guarantee pods to run then scans
+are not executed and reports are missing. Since the Compliance Operator
+is important for ensuring compliance, we should give administrators the
+ability to associate a PriorityClass to the operator. This will ensure
```

Comment:
> Since the Compliance Operator
is important for ensuring compliance, we should give administrators the
ability to associate a PriorityClass to the operator

I would reword this to be less opinionated about importance (example):

Clusters that must have compliance evidence can use pod priority with a `ScanSetting` to ensure scans are executed.